pyproject.toml

```diff
@@ -41,7 +41,9 @@ keywords = [
 [tool.poetry.dependencies]
 python = "^3.9"
 apify-shared = ">=1.1.2"
-httpx = ">=0.25.0"
+# TODO: relax the upper bound once the issue is resolved:
+# https://github.com/apify/apify-client-python/issues/313
+httpx = "~0.27.0"
```

Comment:
If it works with v0.25, we should not make it more strict without a reason. Do we have a reason for that? Otherwise, let's use ">=0.25 < 0.28.0".

Reply:
Done